printer_mods/top_gun_de/mount_for_umbilical/README.md

```diff
@@ -0,0 +1,14 @@
+# Guide for an umbilical with top feed, no cable gland required
+
+I added a toolhead board to my Galileo extruder, and the reduced wire-count suggests to move to an umbilical instead of an energy chain. This mount guides a sleeved wire bundle. My umbilical enters the chamber at the top, through the exhaust opening, and is supported by a gallows. It needed a steady mount at the extruder. 
+
+This has no hardware requirements. I put the wires into a standard polyethylene sleeve to protect the wires.
+
+The toolboard is an EBB36 mounted with https://www.printables.com/model/261263-voron-stealthburner-galileo-ebb36-side-mount
+
+The gallows is https://github.com/top-gun/VoronUsers/tree/master/printer_mods/Galvanic/Bowden_Tube_Guide
```

Comment:
The toolhead board and gallows don't seem to be a required part of this mod, It's probably best not to mention them at all, to avoid confusion.

Reply:
Thank you for your feedback! 

I made some better pictures and replaced the old ones, also included a side- and a rear view.

There is now a description of how this get this guide installed, plus a caution on cable routing. References to the top mounting solution from Galvanic have been removed.